### ISSUE TITLE
Battle screen, the first creature in a combat shouldn't have low morale triggered

### ISSUE THREAD

**Issue by Branikolog** [ihhub/fheroes2#5852]
### Preliminary checks

- [X] I've checked that there aren't [**other open issues**](https://github.com/ihhub/fheroes2/issues?q=is%3Aissue) on the same topic.
- [X] I've checked that this issue is reproducible on the [**latest snapshot build**](https://ihhub.github.io/fheroes2/INSTALL.html#snapshots-latest-builds).

### Platform

Windows

### Describe the bug

In the OG we have the first creature always selected, And it never has low morale triggered on its first turn.
Currently, in fheroes2 first creature can skip its turn because of low morale.

### Save file

None.

**Comment by oleg-derevenetz** [comment]
To be honest, I don't see any logical reason why the first unit should be "special" and not receive a negative moral penalty.

**Comment by LeHerosInconnu** [comment]
Hello @Branikolog and @oleg-derevenetz,

> To be honest, I don't see any logical reason why the first unit should be "special" and not receive a negative moral penalty.

Same for me, if it happens, it happens.
It's a risk to be taken.
And I think I've had this happen before in combats, and even with neutral morale against Bone Dragons. :)
And I think I've seen it before too when watching let's play videos.

**Comment by Branikolog** [comment]
@oleg-derevenetz so do I. Probably, it could be "the fastest troop advantage"? :)
That's the way original game works. Cannot tell it looks like a bug, but could be one more rule, that is different from the H3. :)

**Comment by Branikolog** [comment]
Hi, @LeHerosInconnu ,
I've tested ~40 battles with -3 morale. It had never triggered in the first turn.

**Comment by oleg-derevenetz** [comment]
@LeHerosInconnu Yeah. This means that there is no way to get a first turn against faster unit even if his morale is "Treason", which is not fair.

**Comment by LeHerosInconnu** [comment]
@Branikolog and @oleg-derevenetz.

> Hi, @LeHerosInconnu , I've tested ~40 battles with -3 morale. It had never triggered in the first turn.

You are lucky, :)
I will try to find a let's play video where this happens.

**Comment by zenseii** [comment]
Hi, everyone!
Should this issue still be open?

**Comment by oleg-derevenetz** [comment]
Hi @zenseii I don't know really. On the one hand, there is apparently a difference from the original game. On the other hand, I doubt that in this case it is worth reproducing the behavior of the original game, because it looks unfair due to the reasons stated above. My personal opinion is that we can close this issue for now, but reopen later if there are any really serious arguments in favor of replicating the OG behavior.

**Comment by Branikolog** [comment]
Hi, @zenseii , @oleg-derevenetz 
On the other hand this mechanics belong to the original game. Probably original devs wanted to foster attacking, rather than defending. One more rule, a player should count on while playing. In H2 this could be a bonus for attacker, while H3 has none. But who told, these both games should be the same? They both have their unique rules.
I'm not against adding this, but on the other hand, I'm happy with how the engine works now, because of your invaluable contribution. :)
